### ISSUE TITLE
The plugin `stripe_payment` uses a deprecated version of the Android embedding on 1.1.4

### ISSUE THREAD

**Issue by ceelsoin** [jonasbark/flutter_stripe_payment#330]
When i run build command in release mode i receive the output alert: 

> The plugin `stripe_payment` uses a deprecated version of the
> Android embedding.
> To avoid unexpected runtime failures, or future build failures, try
> to see if this plugin supports the Android V2 embedding. Otherwise,
> consider removing it since a future release of Flutter will remove
> these deprecated APIs.
> If you are plugin author, take a look at the docs for migrating the
> plugin to the V2 embedding:
> https://flutter.dev/go/android-plugin-migration.

My pubspec:
  

> stripe_payment: ^1.1.4

**Comment by arasu33** [comment]
Same issue

**Comment by mk120851** [comment]
Me too

**Comment by julianfrancodev** [comment]
Same issue

**Comment by seanwestfall** [comment]
Want to +1 this, as I have the same build error when running `flutter pub get`.

**Comment by Chiomsyn** [comment]
Is there anyway to resolve this issue?

**Comment by DanielRap** [comment]
this plugin is discontinue, please see
https://github.com/jonasbark/flutter_stripe_payment/blob/master/migration.md

**Comment by seanwestfall** [comment]
Yes, apparently the old `stripe_payment` library has been discontinued and the new one is [`flutter_stripe`](https://pub.dev/packages/flutter_stripe)
